scripts/_common.sh

```diff
@@ -20,7 +20,54 @@ pkg_dependencies="$php_dependencies"
 #=================================================
 # EXPERIMENTAL HELPERS
 #=================================================
+ynh_send_readme_to_admin() {
+	local app_message="${1:-...No specific information...}"
+	local recipients="${2:-root}"
 
+	# Retrieve the email of users
+	find_mails () {
+		local list_mails="$1"
+		local mail
+		local recipients=" "
+		# Read each mail in argument
+		for mail in $list_mails
+		do
+			# Keep root or a real email address as it is
+			if [ "$mail" = "root" ] || echo "$mail" | grep --quiet "@"
+			then
+				recipients="$recipients $mail"
+			else
+				# But replace an user name without a domain after by its email
+				if mail=$(ynh_user_get_info "$mail" "mail" 2> /dev/null)
+				then
+					recipients="$recipients $mail"
+				fi
+			fi
+		done
+		echo "$recipients"
+	}
+	recipients=$(find_mails "$recipients")
+
+	local mail_subject="☁️🆈🅽🅷☁️: \`$app\` has important message for you"
+
+	local mail_message="This is an automated message from your beloved YunoHost server.
+Specific information for the application $app.
+$app_message
+---
+Automatic diagnosis data from YunoHost
+$(yunohost tools diagnosis | grep -B 100 "services:" | sed "/services:/d')"
```

Comment:
This version of ynh_send_readme_to_admin is outdated, because `yunohost tools diagnosis` doesn't exist since ages ... I would look for a newer version in the experimental helpers repo in case a newer one exists

Reply:
Ah ! ok... I have look in the script from cryptpad_ynh, few days ago an upgrade release have send a message... It no more available ?? Also I have added a double quote here https://github.com/YunoHost-Apps/garradin_ynh/commit/267e2c3995d3396eb578038f11a4437820f3225e

Reply:
So, we wait for a future fix... Or another way exist ?

Reply:
@alexAubin I have remove these lines, is it ok ? if I have understand it was just to send a Readme from diagnosis ?

Reply:
> @alexAubin I have remove these lines,
> 
> ```
> Automatic diagnosis data from YunoHost
> $(yunohost tools diagnosis | grep -B 100 "services:" | sed "/services:/d')"
> ```
> 
> is it ok ? if I have understand it was just to send a Readme from diagnosis ?

Sorry, but testing on local virtual yunohost if I remove these lines the upgrade fails. see https://paste.yunohost.org/raw/yidolewaso
If I put them again, it works nice. I can found the mail send only by cli:
```
# cat /var/mail/rodtest/new/1676767705.M961608P21869.yunohost.packaging,S=1435,W=1459
Return-Path: <root@yunohost.packaging>
Delivered-To: rodtest@yunohost.packaging
Received: by yunohost.packaging (Postfix, from userid 0)
	id CBE5E8B59E; Sun, 19 Feb 2023 01:48:25 +0100 (CET)
Content-Type: text/plain; charset=UTF-8
Subject: ☁️🆈🅽🅷☁️: `garradin` has important message for you
To: <root@yunohost.packaging>
X-Mailer: mail (GNU Mailutils 3.10)
Message-Id: <20230219004825.CBE5E8B59E@yunohost.packaging>
Date: Sun, 19 Feb 2023 01:48:25 +0100 (CET)
From: root <root@yunohost.packaging>

This is an automated message from your beloved YunoHost server.
Specific information for the application garradin.
Garradin was successfully upgraded :)
Garradin becomes Paheko. You can now upgrade Garradin with Paheko ! Don't stay with this repository, it will be no more supported.
Take a Look at the Paheko repository and read the instructions how to migrate your application Garradin to Paheko:
https://github.com/YunoHost-Apps/paheko_ynh/
This can only be done from the command-line interface - e.g. through SSH. Once you're connected, you simply have to execute the following:
'yunohost app upgrade garradin -u https://github.com/YunoHost-Apps/paheko_ynh --debug'
The --debug option will let you see the full output. If you encounter any issue, please report it aand paste the logs.
Important: After the migration, you'll have to wait a couple of minutes (at most 3 minutes) before you can start using Paheko.
---
```